### PR TITLE
Bump etcd-druid to v0.15.3

### PR DESCRIPTION
**Pull request by afritzler**
**How to categorize this PR?**

/area dev-productivity
/kind enhancement

**What this PR does / why we need it**:

This PR bumps the `etcd-druid` dependency to v0.15.3.

**Which issue(s) this PR fixes**:
Fixes #

**Release note**:
```other developer
Bump etcd-druid dependency to v0.15.3
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,7 +9,7 @@ require (
 	github.com/bronze1man/yaml2json v0.0.0-20211227013850-8972abeaea25
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/gardener/dependency-watchdog v0.7.0
-	github.com/gardener/etcd-druid v0.15.0
+	github.com/gardener/etcd-druid v0.15.3
```
Comment (acumino) [inline go.mod:12]: https://github.com/gardener/gardener/pull/7248 Already tackles this. And will be merged soon.

Comment (afritzler) [reply go.mod:12]: Oh I guess I missed that! I guess then we can close this one.

**Comment by afritzler** [comment]
Closing in fav of #7248

**Pull request closed without merging**